features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -135,7 +135,8 @@ class MessageComposerPresenter @Inject constructor(
                         Composer(
                             inThread = false,
                             isEditing = composerMode.value is MessageComposerMode.Edit,
-                            isReply = composerMode.value is MessageComposerMode.Reply
+                            isReply = composerMode.value is MessageComposerMode.Reply,
+                            isLocation = false,
```

Comment:
Maybe add a todo for the team working on location?

Reply:
I don't think the composer mode will ever be affected by location. Sending a location is handled in a whole different screen on Android.